tests/e2e/tests/pilot/testdata/v1alpha1/egress-rule-nghttp2.yaml.tmpl

```diff
@@ -1,11 +0,0 @@
-apiVersion: config.istio.io/v1alpha2
```

Comment:
don't you need a replacement rule ? there is already one for google?

Reply:
https://github.com/istio/istio/blob/master/tests/e2e/tests/pilot/testdata/v1alpha1/egress-rule-google.yaml.tmpl